src/features/courses/course-route.ts

```diff
@@ -438,15 +483,24 @@ router.get('/question/:id',
 
         const params = req.params as GetQuestionRequest.params;
         try {
-            // TODO handle not found case
-            const question = await courseController.getQuestion({
-                questionId: params.id,
-                userId: session.userId,
-                formURL: req.originalUrl,
-                role: user.roleId,
-                readonly: req.query.readonly,
-                workbookId: req.query.workbookId,
-            });
+            let question;
+            if (req.query.userId) {
```

Comment:
Instead of relying on the userId we should probably add a query flag to fetch from renderer

Reply:
Nit ^

Reply:
Talked to @PanopticaRising not doing this now